### DISCUSSION TITLE
SSD202D support Lineout Audio PlayBack?

### DISCUSSION THREAD

**Discussion by xinyu-zhu** [linux-chenxing/linux-chenxing.org#65] [General]
Hello! I saw this video on twitter[https://twitter.com/linux_chenxing/status/1504389676861628421].
I wonder if ssd202d now supports audio playback ?
If yes, where can I find the corresponding code?

Thank You~

**Comment by fifteenhex** [comment]
It works enough for standard apps.

My code is here: https://github.com/linux-chenxing/linux/tree/mstar_v5_19_rebase/sound/soc/mstar

**Reply by xinyu-zhu** [reply]
Tkank you ^-^
